nano-service.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.required_ruby_version = ">= #{IO.read(File.expand_path('.ruby-version', __dir__)).chomp}"
 
-  s.add_dependency 'activerecord',  '>= 6.0.4', '< 7.1.0'
-  s.add_dependency 'activesupport', '>= 6.0.4', '< 7'
-  s.add_dependency 'globalid',      '~> 1.0', '>= 1.0.1'
```

Comment:
Huh, what does this file do? I've not seen it before.

Reply:
The .gemspec file? It's used for packaging the gem if you want to host it somewhere or want to install via git in bundler. https://guides.rubygems.org/make-your-own-gem/#your-first-gem -- this is mostly used to convey dependencies and version constraints to whatever is consuming the gem/repo.